vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -1070,7 +1074,7 @@ public Component getListCellRendererComponent(
    * @author rkinney
    *
    */
-  private static class TraitClipboard {
+  protected static class TraitClipboard {
```

Comment:
Is this actually necessary?

Reply:
Yes, because MPL's subclass of PieceDefiner now accesses this class to check the type of the copied piece. It would not compile unless I changed this.

Reply:
Oh, I see what's going on now. `clipBoard` is a `protected` field in the base class, but its _type_ had been `private` so you couldn't see the class definition in order to call any methods on it.